app/assets/stylesheets/govuk-component/govspeak/_steps.scss

```diff
@@ -9,15 +12,27 @@
     list-style-type: decimal;
     margin-left: 0;
     padding: 0.75em 0 0.75em 2.2em;
+    position: relative;
 
-    @for $i from 1 through 14 {
-      &:nth-child(#{$i}) {
-        background-image: image-url("icon-steps/icon-step-#{$i}.png");
+    &:before {
+      position: absolute;
+      left: 0;
+      margin-top: -1px;
+      height: 18px;
+      width: 18px;
+      padding: 4px;
+      border-radius: 18px;
+      background-color: $black;
+      color: $white;
+      @include core-16();
+      font-weight: bold;
+      text-align: center;
+    }
 
-        @include device-pixel-ratio() {
-          background-image: image-url("icon-steps/icon-step-#{$i}-2x.png");
-          background-size: 24px 24px;
-        }
+    @for $i from 1 through 14 {
```

Comment:
yeah we had a little discussion about that, did you see the extra line of css i added? i need to be able to get at each one explcitity with `step-step1`